File: tempo-jogo.py
hora_inicial, hora_termino = map(int, input().split())

try:
    tempo = 0
    if (hora_inicial >= 0 and hora_inicial <= 23 and hora_termino >= 0 and hora_termino <= 23):
        if (hora_inicial > hora_termino):
            tempo = 24 - hora_inicial + hora_termino
        elif (hora_inicial < hora_termino):
            tempo = hora_termino - hora_inicial
        elif (hora_inicial == hora_termino):
            tempo = 24

        print('O JOGO DUROU {} HORA(S)'.format(tempo))
except:
    pass

File: tempo-jogo.c
#include <stdio.h>

int main()
{
    int hora_inicial, minuto_inicial, hora_final, minuto_final, tempo, tempo_hora, tempo_minuto;

    scanf("%d %d %d %d", &hora_inicial, &minuto_inicial, &hora_final, &minuto_final);

    tempo = (((hora_final * 60) + minuto_final) - ((hora_inicial * 60) + minuto_inicial));

    if (tempo <= 0)
    {
        tempo = tempo + (24 * 60);
    }

    tempo_hora = tempo / 60;
    tempo_minuto = tempo % 60;

    printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", tempo_hora, tempo_minuto);

    return 0;
}
